Lib/pdb.py

```diff
@@ -945,7 +951,8 @@ def do_ignore(self, arg):
     complete_ignore = _complete_bpnumber
 
     def do_clear(self, arg):
-        """cl(ear) filename:lineno\ncl(ear) [bpnumber [bpnumber...]]
+        """cl(ear) [filename:lineno | bpnumber ...]
```

Comment:
This seems like a more accurate description, right?

```suggestion
        """cl(ear) [filename:lineno] [bpnumber ...]
```

Reply:
`clear` command takes either a `filename:lineno` or a list of `bpnumber`. `[filename:lineno] [bpnumber ...]` suggests that they could both exist, which won't work.

Reply:
Ah, I missed the early return on 990!

Reply:
Actually, now that I took at look at the code, this is not what I would write. I'd use three mutual exclusive if cases instead of early returns.